libcudacxx/test/libcudacxx/std/utilities/meta/meta.trans/meta.trans.other/result_of.fail.cpp

```diff
@@ -8,6 +8,7 @@
 
 // Mandates: invoke result must fail to compile when used with device lambdas.
 // UNSUPPORTED: clang && (!nvcc)
+// UNSUPPORTED: nvcc-12.3, nvcc-12.4
```

Comment:
suggestion: Upon review, this test needs to be modified a bit. We still want to ensure that using `invoke_result` on an extended lambda _without_ a trailing return type still fails. 

So I suggest this test can be minimally modified by just removing the trailing return types on likes 36 & 37. Then you don't need to add any new `UNSUPPORTED` cases.

Reply:
Fixed